.github/workflows/config.yml

```diff
@@ -42,3 +42,21 @@ jobs:
         env:
           SWIFT_SUFFIX: ''
           SAMPLE: Config
+      - name: Build Catalyst
+        run: |
+          cd config
+          ../scripts/install_prereqs/config.sh
+          pod install
+          xcodebuild \
```

Comment:
We need to add something like the firebase-ios-sdk's build.sh script for managing xcodebuild options, but that can be done separately.